lib/mumukit/bridge.rb

```diff
@@ -1,6 +1,13 @@
 require 'mumukit/bridge/version'
+require 'mumukit/bridge/response_type'
+require 'mumukit/bridge/array'
+require 'mumukit/bridge/boolean'
+
 require 'rest_client'
 
+require 'active_support/hash_with_indifferent_access'
+require 'active_support/core_ext/object'
```

Comment:
Nice catch. `hash_with_indifferent_access` was not being used. 

`core_ext` are used in `present?` methods